pkg/controller/plan/kubevirt.go

```diff
@@ -1062,7 +1062,7 @@ func (r *KubeVirt) virtualMachine(vm *plan.VMStatus) (object *cnv.VirtualMachine
 	//convention it will be automatically changed.
 	var originalName string
 
-	if errs := k8svalidation.IsDNS1123Label(vm.Name); len(errs) > 0 {
+	if errs := k8svalidation.IsDNS1123Label(vm.Name); !r.Plan.IsSourceProviderOCP() && len(errs) > 0 {
```

Comment:
how can it be that we get an error from `IsDNS1123Label` when `IsSourceProviderOCP()==true`?